### PR TITLE
update lib and integrate qunit-test-runner

### PR DESCRIPTION
**Pull request by btamas**
Update tao-core-libs and remove local libs.
Also integrato tao-qunit-testrunner

Requirement:
- [ ] https://github.com/oat-sa/tao-core-libs-fe/pull/6
- [ ] Update tao-core-libs version in package.json
- [ ] Update package-lock.json

### CODE REVIEW

File: merge.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+cp -r ui_should/$1 src
```
Comment (krampstudio) [inline merge.sh]: I think those files shouldn't have been commited

File: fileMerge.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+npx amdtoes6 ui_should/$1.js > src/$1.js && rm ui_should/$1.js
```
Comment (krampstudio) [inline fileMerge.sh]: this file seems to be external tooling

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
+FROM node:8-slim
```
Comment (krampstudio) [inline Dockerfile]: I think we should rely on only one image instead of maintaining Dockerfile in every repo

Comment (btamas) [reply Dockerfile]: These files are unfortunately committed files. Thanks to notice!

**Pull request merged**